src/mono/wasm/runtime-test.js

```diff
@@ -333,24 +344,23 @@ async function loadDotnet(file) {
             const script = document.createElement("script");
             script.src = file;
             document.head.appendChild(script);
-            return globalThis.Module;
+            while (true) {
+                if (globalThis.createDotnetRuntime) {
+                    return globalThis.createDotnetRuntime;
+                }
+                await new Promise(resolve => setTimeout(resolve, 10));
```

Comment:
Is this basically "loading started happening asynchronously at some point, and it will finish at some point but we're not sure when, so spin and check for it to finish every 10ms"? A comment explaining this loop would be good